main.nf

```diff
@@ -312,28 +325,26 @@ process RealignerTargetCreator {
   """
 }
 
-verbose ? intervals = intervals.view {"Intervals to phase: $it"} : ''
+if (verbose) intervals = intervals.view {"Intervals to phase: $it"}
 
 bamsAndIntervals = duplicatesRealign
   .phase(intervals)
-  .map{duplicatesRealign, intervals ->
+  .map{dr, i ->
```

Comment:
I agree that all your changes go towards an improved readability, except for this one.
Does it really worth it to replacing `duplicatesRealign, intervals` by `dr, i` ?